tests/format/typescript/type-alias/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,5 +1,81 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`conditional.ts format 1`] = `
+====================================options=====================================
+parsers: ["typescript"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+type FallbackFlags<F extends Flags | undefined> =
+  Equals<NonNullableFlag<F>["flags"], {}>    extends   true
+    ? Dict<any>
+    : NonNullableFlag<F>["flags"];
+
+export type UnPromise<Type extends Promise<unknown>> =
+  Type extends Promise<infer Generic>
+    ? Generic
+    : never;
+
+export type Equals<X, Y> =
+  (<T>() => T extends X ? 1 : 2) extends
+  (<T>() => T extends Y ? 1 : 2)
+    ? true
+    : false;
+
+export type _Repeat<A extends any, N extends number, L extends List = []> =
+  __Repeat<N, A, L> extends infer X
+  ? Cast<X, List>
+  : never
+
+export type Repeat<A extends any, N extends number, L extends List = []> =
+  N extends unknown
+  ? L extends unknown
+    ? _Repeat<A, N, L>
+    : never
+  : never
+
+export type Intersect<U1 extends any, U2 extends any> =
+  U1 extends unknown
+  ? U2 extends unknown
+    ? {1: U1, 0: never}[Equals<U1, U2>]
+    : never
+  : never
+
+=====================================output=====================================
+type FallbackFlags<F extends Flags | undefined> =
+  Equals<NonNullableFlag<F>["flags"], {}> extends true
+    ? Dict<any>
+    : NonNullableFlag<F>["flags"];
+
+export type UnPromise<Type extends Promise<unknown>> =
+  Type extends Promise<infer Generic> ? Generic : never;
+
+export type Equals<X, Y> =
+  (<T>() => T extends X ? 1 : 2) extends <T>() => T extends Y ? 1 : 2
+    ? true
+    : false;
```

Comment:
Stable Prettier doesn't format these fine to me. [Playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEMCeAHOACAYgQwBtCAjfMAa10PwHMBnAHl2zgA94oATevGh7AB9sAV25wAZgEsocLgD5sAXgA6UbNgCiARxFEmAOWgGRxfCUJxqdZvIDaKkBP71HAXQA02YAF9F7Th5sGAAnETg1DQ0AfmwAESkwGEZ8KDR5SKikbCMoEzMLK35bBycXdwBuNTV2DAgQmGDMHABVKAAFEIgAWyl6OEYAFWbWDgQgzp6+gbEKKAgAdyh5RVV1bGGsUcDeSd7+xhkJOBDsAHEEE8TFWIvZEMTsbNkANxOqqBq2OoamrZ09IQmAANLwATVWmQAFEN5FCAJTKRSDbbjXjA7CxACMT2wACZEQE0dhYQikRtUdxeGDMdgcdk8ZkYsEwhF1hpshJ9HAKiAPCAIBgYFJoPRkKB8CEugt2pKEGKUEQFvg0GL+SQQuQKHAYABlfDdOAAGRkcGQXKBcHVmsoOt1GHIMloyFC4X5-V6LtZ-NqV0NsCIgxOUElUjgCot-XdTssAEURBB4ObufyAFb0Ni6mNweOJs1ISNWkC6POTDAKkD4egAWlkcjkfNQmqkhCdAGEet18MhK8RG-RswBBGChKQkETwdonE2yZOW-kACxg3UIAHUF1J4PQHWA4Lr5ZupC9N2ge2B6GqQC9wgBJcSwXVgB5CwfcXXoSxzqMgDBdfqrzUMB7X9wxON5GxkfoGk6Oguy-IsHRCKCe3QLB6CfKQhUbX8ZBgVcpC4GAF2QAAOAAGfkQjgXQpComDaDggsU1Qcx8MI4ikDxfkRH6QZzAjZi4G6Eg5C4OQjVSWg9FoKx6i7EcnR7fAJwgEAfB8IA)
This PR addresses assignment layout for them.